.travis.yml

```diff
@@ -22,3 +22,4 @@ script:
   - cd uint/ && cargo test --features=std,quickcheck --release && cd ..
   - cd plain_hasher/ && cargo test --no-default-features && cd ..
   - cd parity-util-mem/ && cargo test --features=estimate-heapsize && cd ..
+  - cd parity-crypto/ && cargo test --all-features && cd ..
```

Comment:
I'm not sure how this works. Error conversion from ring [was removed](https://github.com/paritytech/parity-common/pull/144/files#diff-c6b3ae839850070966d556586872155dL107), but it's used e.g. [here](https://github.com/paritytech/parity-common/pull/144/files#diff-89cede331f3b34fa96c6102348e03a44R34) if I'm not mistaken. 
Also running this command locally doesn't seem to run aes_gcm tests.

Reply:
Oh, interesting.

Reply:
This line is now not needed.